<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="MainWidget" name="openGLWidget_2"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>GroupBox</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>CheckBox</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_2">
               <property name="text">
                <string>CheckBox</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_boneRotations">
            <property name="title">
             <string>Bone Rotations</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="3" column="0" colspan="2">
              <widget class="QSlider" name="horizontalSlider_yRot">
               <property name="minimum">
                <number>-180</number>
               </property>
               <property name="maximum">
                <number>180</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_yRot">
               <property name="text">
                <string>y-Rot:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_zRot">
               <property name="text">
                <string>z-Rot:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QSpinBox" name="spinBox_zRot">
               <property name="minimum">
                <number>-180</number>
               </property>
               <property name="maximum">
                <number>180</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_xRot">
               <property name="text">
                <string>x-Rot:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_xRot">
               <property name="minimum">
                <number>-180</number>
               </property>
               <property name="maximum">
                <number>180</number>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QSlider" name="horizontalSlider_zRot">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <number>-180</number>
               </property>
               <property name="maximum">
                <number>180</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBox_yRot">
               <property name="minimum">
                <number>-180</number>
               </property>
               <property name="maximum">
                <number>180</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QSlider" name="horizontalSlider_xRot">
               <property name="minimum">
                <number>-180</number>
               </property>
               <property name="maximum">
                <number>180</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_render">
            <property name="title">
             <string>Render</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="checkBox_axes">
               <property name="text">
                <string>Axes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_model">
               <property name="text">
                <string>Model</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_skeleton">
               <property name="text">
                <string>Skeleton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_quaternion">
               <property name="text">
                <string>Quaternion</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>main_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_xRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_xRot</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_yRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_yRot</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_zRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_zRot</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_xRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_xRot</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_yRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_yRot</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_zRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_zRot</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
